src/send/test.rs

```diff
@@ -9,10 +9,11 @@ use rocket::{
 };
 use serde_json;
 
-use app_errors::{self, ApplicationError};
+use app_errors;
 use auth_db::DbClient;
 use bounces::Bounces;
 use message_data::MessageData;
+use error::{HandlerErrorKind};
```

Comment:
do we need alphabetical order for this?

Reply:
This is still WIP, so I haven't use `cargo fmt -- --check` yet, that will tell me about this kind of formatting errors.